airflow/opt/providers/etna/etna/tests/test_box_helpers.py

```diff
@@ -0,0 +1,160 @@
+# import re
```

Comment:
Blah... yeah, just blah about these tests.

It's tough to handle both FTP and HTTP requests, so I may wind up not bothering with these, if I can't figure out the issue with the DB connection. Would be nice to at least test some of the non-connection-related logic...

Reply:
Yeah, this is gonna be hard.  I too have struggled to write good tests because this code strikes at a crucial integration point, and various mocking libraries / recording (VCR) do not handle correctly all the socket level stuff going on here.

Reply:
If we have to, I'm ok saying that we have a manual, smoke screen tests, a script that we run by hand that executes some tests without recordings, given some inputs from the runner, to verify integration of some parts of the system.  Again, not ideal, but better than no testing.

This level of integrations tooling gets tricky to test overall, and is a rabbit hole.  Let's be pragmatic and aim for value vs time here!